src/main/java/seedu/duke/commands/HelpCommand.java

```diff
@@ -11,8 +12,23 @@
 import java.lang.reflect.InvocationTargetException;
 
 public class HelpCommand extends Command {
+    boolean runTime = false;
 
     public HelpCommand() {
+        syntax = "help";
+    }
+
+    public HelpCommand(String input) throws
+            HelpException {
+        if (input.equals("help rt")) {
```

Comment:
So different constructors will be called depending on the runtime mode?

Reply:
Yes, it'll be toggled with `help rt`. I implemented this so it's easier to integrate the runtime help (for devs) into the help command (for users).

In other words, the current process for updating the help command would be:
1. Dev runs `help rt`
2. Dev runs `help` to get all command syntaxs at runtime (since intellij supports this, but doesn't work on a <kbd>.Jar</kbd> file
3. Dev copies the updated help command (sorted alphabetically, with syntax extracted at runtime)
4. Dev pastes the updated help command string into command constants

This implementation removes the need for uncommenting/ commenting the `ClassPackageReader` implementation I'm doing now with help, although there's still a need to "hard-code" the help message string for the user's help command.

Reply:
Thanks for the question, I think this is an important issue to be addressed in the developer guide too!